### PR TITLE
FOGL-8587: GUI support for object type kvlist config item

### PR DESCRIPTION
**Pull request by hemantgarg923**

### CODE REVIEW

File: src/app/components/core/configuration-manager/kv-list-type-configuration/kv-list-type-configuration.component.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { ChangeDetectorRef, Component, EventEmitter, Input, OnInit, Output } fro
 import { FormArray, FormBuilder, FormGroup, FormGroupDirective, Validators } from '@angular/forms';
```
Comment (MonikaSharma06) [inline src/app/components/core/configuration-manager/kv-list-type-configuration/kv-list-type-configuration.component.ts:2]: remove unused import `FormGroupDirective`

File: src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts [hunk 2]
```diff
@@ -92,9 +93,13 @@ export class ConfigurationGroupComponent implements AfterViewInit {
...
         listConfig.push(this.category.config[k]);
       }
+      else if(this.category.config[k].type == 'kvlist') {
+        this.category.config[k].key = k;
+        kvlistConfig.push(this.category.config[k]);
+      }
       this.category.config[k].key = k;
       return this.category.config[k];
-    }).filter(obj => !(obj.readonly || obj.type == 'bucket' || obj.type == 'list')); // remove readonly, type=bucket and type=list from config array
+    }).filter(obj => !(obj.readonly || ['bucket', 'list', 'kvlist'].includes(obj.type))); // remove readonly, type=bucket, type=list and type=kvlist from config array
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts:102]: This readonly is being fixed in 2.5.0rc.

File: src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts [hunk 3]
```diff
@@ -118,6 +123,15 @@ export class ConfigurationGroupComponent implements AfterViewInit {
         this.groups.push({ category: this.category.name, group: (config.displayName ? config.displayName : config.description), config: config, type: config.type, key: config.key });
       });
     }
+
+    if (kvlistConfig.length > 0) {
+      kvlistConfig.forEach(config => {
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts:128]: Do we need length check? Or not null/undefined

ForEach will run if length only

File: src/app/components/core/configuration-manager/kv-list-type-configuration/kv-list-type-configuration.component.css [hunk 4]
```diff
@@ -2,3 +2,20 @@
...
+}
+
+.kvlist-divider {
+  background-color: white;
+  border: 1px dashed whitesmoke;
+  display: block;
+  margin: 0.5rem 0 0 0;
+}
+
+.info-icon {
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/kv-list-type-configuration/kv-list-type-configuration.component.css:19]: Global 

Use bootstrap icon for info throughout, including other products?

**Pull request merged**